packages/cli/src/lib/oauth/oauth.ts

```diff
@@ -54,7 +54,9 @@ export class OAuth {
 
   private openLoginPage(state: string) {
     const url = this.buildAuthorizationUrl(state);
-    opener(url);
+    open(url).then((browserProcess) => {
```

Comment:
why not use await?

Reply:
Because we don't need for the browser to be open and all to start opening the 'listening' server, so we might as well do so to shave some time

Reply:
Another alternative would be to change the `openLoginPage` to be asynchronous and thus await on the `open` call but not on the call of `openLoginPage`.

What do you prefer?